### ISSUE TITLE
Changing browser tab clear drawing buffer.

### ISSUE THREAD

**Issue by Kurtil** [pixijs/pixi.js#7154]
[jsfiddle example here.](https://jsfiddle.net/GaetanLagier/q5wt8rns/13/)

Quick preview:
![pixiBug](https://user-images.githubusercontent.com/22523482/104629990-80c0bf80-569a-11eb-959b-29e7b9f47525.gif)

### Steps to reproduce

- Initial state:
<img width="428" alt="Screenshot 2021-01-14 at 18 59 14" src="https://user-images.githubusercontent.com/22523482/104630069-9cc46100-569a-11eb-86c7-cc66a6fa268c.png">

- Click on "Draw from canvas 1 to canvas 2". This uses 2d context of canvas 2 to `drawImage`:

```javascript
const ctx = canvas2.getContext("2d");
ctx.drawImage(canvas1, 0, 0);
```
<img width="421" alt="Screenshot 2021-01-14 at 19 01 43" src="https://user-images.githubusercontent.com/22523482/104630365-f4fb6300-569a-11eb-9c4c-d99165a84be9.png">
Note: it works only if `preserveDrawingBuffer` is set to `true` on the canvas1 PIXI.Renderer.

- Change tab.

- Come back.

- Click again on "Draw from canvas 1 to canvas 2". The red square disappear.

<img width="429" alt="Screenshot 2021-01-14 at 19 04 08" src="https://user-images.githubusercontent.com/22523482/104630642-4ad00b00-569b-11eb-99d3-b85c23a1f909.png">

### Expected Behavior
The red square must not disappear.

### Current Behavior
The red square disappears.

Chrome ONLY.

### Does not work on:
Chrome 87. (Windows 10, Linux UBUNTU 20.04, MacOS 10.15.7)
PIXI v5.X v6.X

Changing window on Windows and MacOS also leads to the issue.

### Works on:
Firefox (Windows 10, Linux UBUNTU 20.04, MacOS 10.15.7.)
PIXI v5.X v6.X

**Comment by bigtimebuddy** [comment]
Good bug!

Here's a webgl example _without_ PIXI: https://jsfiddle.net/bigtimebuddy/s418rk6y/

This seems like a browser bug (not PixiJS) and you might want to report to Chrome/Chromium.

**Comment by Kurtil** [comment]
Thanks!
https://bugs.chromium.org/p/chromium/issues/detail?id=1166789

**Comment by bigtimebuddy** [comment]
I'm going to close this bug because I don't think there anything we can do to mitigate or workaround this. Likely will need to be fixed in the browser.

**Comment by Kurtil** [comment]
The original chromium issue was merged into another one: https://bugs.chromium.org/p/chromium/issues/detail?id=736969
The fix is shipped with chromium M-90 # 90.0.4395.0
I can no longer reproduce it! 🎉 

Issue #7153 seems to also be fixed thanks to this release! 🎉 x2 !!

**Comment by bigtimebuddy** [comment]
Thanks for the update @Kurtil
